libs/ui-lib/lib/common/config/constants.ts

```diff
@@ -33,6 +34,15 @@ export const hostRoles = (t: TFunction): HostRole[] => [
       'ai:Runs application workloads. Connect at least 5 hosts to enable dedicated workers.',
     ),
   },
+  ...(isInOcm && !isSNO
```

Comment:
we actually do not have to pass the `isInOcm` prop all the way down. We can just import it from `common/api/axiosClient.ts`


about the `isSNO` prop - the ability to change the host role is driven by `canEditRole` function. It already ensures that role is not changeable for sno clusters. We should not introduce another way of disabling the role selection.